pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -161,44 +135,33 @@ class SELinuxEventDetails extends React.Component {
                         lastCommand = false;
                     }
                 });
-                doElement = parts.map(p => p[0] == "#" ? <pre key={p}>{p.substr(2)}</pre> : <span key={p}>{p}</span>);
+                doElement = parts.map(p => p[0] == "#" ? <TextArea isReadOnly key={p}>{p.substr(2)}</TextArea> : <span key={p}>{p}</span>);
             }
 
-            var detailsLink = <Button variant="link" isInline onClick={ self.handleSolutionDetailsClick.bind(self, itmIdx) }>{ _("solution details") }</Button>;
-            var doState;
-            var doElem;
-            var caret;
-
-            if (self.state.solutionExpanded[itmIdx]) {
-                caret = <i className="fa fa-angle-down" />;
-                doState = <div>{caret} {detailsLink}</div>;
-                doElem = doElement;
-            } else {
-                caret = <i className="fa fa-angle-right" />;
-                doState = <div>{caret} {detailsLink}</div>;
-                doElem = null;
-            }
             return (
-                <div className="list-group-item selinux-details" key={itm.analysisId + (itm.ifText || "") + (itm.doText || "")}>
-                    <div>
-                        <div>
-                            <span>{itm.ifText}</span>
-                        </div>
+                <Card isCompact key={itm.analysisId + (itm.ifText || "") + (itm.doText || "")}>
```

Comment:
I am a bit on fence here. When I opened card that had just one solution I found it weird that it contains card. I came here to comment on it. But then I opened some other report that had two solutions and I saw why you picked it.

![solutions](https://user-images.githubusercontent.com/12330670/110683844-c900e780-81dc-11eb-9209-5a7117f8c465.png)

But I lean a bit more towards that Card is not the right component, it feels like it does not belong there. Have you thought about [DataList](https://www.patternfly.org/v4/components/data-list/)? Or just keeping it as `div`s and smashing [Divider](https://www.patternfly.org/v4/components/divider) inbetween?

Reply:
![Screen Shot 2021-03-11 at 10 35 04](https://user-images.githubusercontent.com/14921356/110766447-76afdd00-8255-11eb-8e97-9e8869c40442.png)